charts/amazon-network-policy-controller-k8s/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
```

Comment:
last nit, otherwise I think this is good to go after rebasing: shouldn't this be `v1`? I remember we had a problem with the EFA plugin helm chart where we had to go with v1: https://github.com/aws/eks-charts/blob/master/stable/aws-efa-k8s-device-plugin/Chart.yaml#L1

Reply:
I'm curious to know the limitation we faced there. I tried deploying the helm chart in my cluster with v2, I didn't run into any issues. Even CNI migrated to v2 last year

Reply:
It looks like CNI still uses v1: https://github.com/aws/amazon-vpc-cni-k8s/blob/master/charts/aws-vpc-cni/Chart.yaml#L1

I vaguely recall it having to do with a helmv2 incompatibility. This may be fine here as this chart only claims to support helmv3

Reply:
Ah sorry, I got confused with helm v2 to v3 migration. We shouldn't be affected by these changes from v1 to v2


>Changes from v1 to v2:
>A dependencies field defining chart dependencies, which were located in a separate requirements.yaml file for v1 charts (see [Chart Dependencies](https://helm.sh/docs/topics/charts/#chart-dependencies)).
>The type field, discriminating application and library charts (see [Chart Types]
(https://helm.sh/docs/topics/charts/#chart-types)).